static/app/views/replays/details.tsx

```diff
@@ -105,40 +90,21 @@ class ReplayDetails extends AsyncView<Props, State> {
                 {label: t('Replay Details')}, // TODO: put replay ID or something here
               ]}
             />
-            <EventHeader event={event} />
+            <TitleWrapper>Replay: {this.props.params.replayId}</TitleWrapper>
           </Layout.HeaderContent>
         </Layout.Header>
 
-        <Layout.Body>
-          <Layout.Main>
-            <RRWebIntegration event={event} orgId={orgSlug} projectId={projectSlug} />
-          </Layout.Main>
-          <Layout.Side>
-            <TagsTable
-              generateUrl={this.generateTagUrl}
-              event={event}
-              query={eventView.query}
-            />
-          </Layout.Side>
-        </Layout.Body>
+        <ReplayEvents {...this.props} eventSlugs={eventSlugs} />
```

Comment:
Notice we are passing `eventSlugs` into `<ReplayEvents>`. This details component is loading the list of `event ids` related to the given `replayId`, and then the sub-component will load the event details for each event